docs/github-merge-strategy.md

```diff
@@ -6,6 +6,46 @@ This document describes the merge strategy used for Pull Requests within all rep
 
 Every Pull Request that is made to an InstructLab repository should meet the below requirements - certain repositories such as [taxonomy](https://github.com/instruct-lab/taxonomy) may have additional requirements.
 
+### Commit Messages and PR Description
+
+PRs should be formatted as a series of logical commits that implement the
+overall change proposed in the PR. Each commit should represent a logical step
+in the progression of the change. The commit message for each should clearly
+explain that step of the progression.
+
+Here are a few examples of PRs that contained a series of commits:
+
+- <https://github.com/instructlab/instructlab-bot/pull/125/commits>
+- <https://github.com/instructlab/instructlab/pull/994/commits>
+- <https://github.com/instructlab/instructlab/pull/951/commits>
+
+Here are some external resources that provide guidance on writing good commit messages:
+
+- <https://www.berrange.com/tags/commit-message/>
+- <https://docs.kernel.org/process/submitting-patches.html#separate-your-changes>
+- <https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#smaller-is-better-small-commits-small-pull-requests>
+
+While not a requirement right now, we may want to consider following a spec like
```

Comment:
Perhaps we can also mention 50/72 rule, since Conventional Commits don't seem to cover it. E.g. this: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html